### DISCUSSION TITLE
setText connect and setText disconnect instead of checked and unchecked for the connect option, like teamtalk classic

### DISCUSSION THREAD

**Discussion by peter1384** [BearWare/TeamTalk5#2315] [Ideas]
@CoBC currently, the connect option in the client menu is set as checked when connected, and unchecked when not connected, but like teamtalk classic, it would be a bit better and more understandable if the lable changes between connect and disconnect, while the shortcut remanes f2 always, like teamtalk classic again.

**Comment by CoBC** [comment]
As I said previously, you don't have to mention me in each comment. All 
users should participate to discussion, it's exactly the goal of 
discussion. Thanks for your understanding

**Comment by peter1384** [comment]
ok sorry, I just did that because my face was on you to talk to you, sorry.
